interface/patient_file/report/patient_report.php

```diff
@@ -386,6 +387,11 @@ function show_date_fun(){
                                                 " value='" . $result{"encounter"} . "'" .
                                                 " class='encounter_form' ".
                                                 ">" . xl_form_title($result{"form_name"}) . "<br>\n");
+		*/
+		 echo "<input type='checkbox' name='".$result['formdir']."_".$result['form_id']."'".
```

Comment:
I went back into the program and randomly selected patients from the calendar. It is across the board that the first encounter does not display any form that was filled out on that visit. 

Yes, I agree that they original writer went through a lot of hoops to make sure that the form name matches the registry.

The thing that bothers me now that I am taking a second look past the point of change is that the next foreach loop has  if ($toprint = isset($html_strings[$var])) shouldn't it be a == instead?

But after testing the == it  had no affect on the out come.